model/host/db.go

```diff
@@ -225,9 +225,11 @@ func startedTaskHostsQuery(distroID string) bson.M {
 
 func idleHostsQuery(distroID string) bson.M {
 	query := bson.M{
-		RunningTaskKey: bson.M{"$exists": false},
-		StatusKey:      evergreen.HostRunning,
-		StartedByKey:   evergreen.User,
+		StartedByKey:     evergreen.User,
+		ProviderKey:      bson.M{"$in": evergreen.ProviderSpawnable},
+		RunningTaskKey:   bson.M{"$exists": false},
+		HasContainersKey: bson.M{"$ne": true},
```

Comment:
Since this doesn't run for container parents, is it supposed to also filter out containers (i.e. is this supposed to exclude container hosts that set `ParentID`)?

Reply:
I copied this from `IdleEphemeralGroupedByDistroID` because I want to only run this on hosts that idle termination would run on. I think I would want to exclude container parents, but terminating individual containers should be fine (I assume that ParentID refers to that host's parent, and therefore that container hosts with `ParentID` set are children container task hosts running on a parent host)

Reply:
Ok, if you want to also terminate idle containers, then this is fine.